context.go

```diff
@@ -100,8 +100,8 @@ func (p *Project) NewContext(opts ...func(*Context) error) (*Context, error) {
 
 	// backfill enbedded go/build.Context
 	ctx.Context = &build.Context{
-		GOARCH:   ctx.gotargetos,
-		GOOS:     ctx.gotargetarch,
+		GOARCH:   ctx.gotargetarch,
```

Comment:
please swap the order, goos then goarch, that's why I made the mistake in the first place.